auto_submit/lib/service/approver_service.dart

```diff
@@ -71,7 +71,7 @@ class ApproverService {
 
     final gh.CreatePullRequestReview review =
         gh.CreatePullRequestReview(slug.owner, slug.name, pullRequest.number!, 'APPROVE');
-    botClient.pullRequests.createReview(slug, review);
+    await botClient.pullRequests.createReview(slug, review);
```

Comment:
Can we add https://dart-lang.github.io/linter/lints/unawaited_futures.html to the main analysis options?

These types of issues are hard to debug, but can be easily prevented